CMakeLists.txt

```
@@ -466,6 +473,11 @@ if(MIOPEN_USE_ROCBLAS)
     set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocblas")
 endif()
 
+if(BUILD_COMPILER_PATHS_V2)
+    message(STATUS "Package Dependency with rocm-core for getROCmInstallPath() API")
+    set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocm-core")
+endif()
```

Message is not necessary, we already have one.
```suggestion
if(MIOPEN_OFFLINE_COMPILER_PATHS_V2)
    set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocm-core")
endif()
```